src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1417,6 +1486,52 @@ export const handleGetSettings: ControllerHandler<
     })
 }
 
+/**
+ * Handler for GET api/platform/v1/admin/forms/:formId/webhookSettings.
+ *
+ * @returns 200 with latest webhook and response mode settings
+ * @returns 401 when current user is not logged in
+ * @returns 403 when current user does not have permissions to obtain form settings
+ * @returns 404 when form to retrieve settings for cannot be found
+ * @returns 409 when saving form settings incurs a conflict in the database
```

Comment:
for GET request, there should not be 409 (no saving form settings) 😅

Reply:
heh thanks, deleted in [9cdd65b](https://github.com/opengovsg/FormSG/pull/6608/commits/9cdd65b29676ff8e77cd2edbe08c7d36c5b3af03)